website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,70 @@
+
+After a few months, the fundraising efforts allowed paying Henry every month: we settled on \$11,000 per month as a baseline salary for working full-time on open source. We had a large grant (\$10k for 10 months) from [Handshake](https://handshake.org/grant-sponsors/) which initially helped boosting up our funds, but even when it ended we didn't have problems paying our team thanks to some amazing sponsors such as Airbnb, Trivago, Gatsby, AMP, and Salesforce.
+
+We didn't want to stop there. We thought that by demonstrating our ability to create and _pay_ a strong team, more features and improvements (and in turn, value) would be delivered via Babel, which would continue the momentum of funding and sponsorship. We decided to start paying [Jùnliàng](https://twitter.com/jlhwung), [Nicolò](https://twitter.com/NicoloRibaudo) and [Kai](https://twitter.com/kai_cataldo) a "part-time" rate: we could initially afford \$2,000 USD per month. Our hope was by [announcing this plan](https://babeljs.io/blog/2019/11/08/babels-funding-plans) and continuing our fundraising efforts, we would be able to increase the budget and raise them to the full-time rate.
+
+## What Babel is today
+
+It has been more than a year since then and we've both done and learned a lot.
+
+[We've implemented](https://babel.dev/docs/en/features-timeline) support for many new ECMAScript proposals, kept up with every new TypeScript and Flow release, and designed [new features](https://babel.dev/assumptions) to allow producing smaller compiled output. We are now nearing the next major release, Babel 8. 
+
+Babel is continued to be used by thousands of [companies all over the world](https://babeljs.io/en/users.html). It's integrated into all kinds of frameworks in the JavaScript ecosystem, whether it's React, Next.js, Vue, Ember, Angular, etc. We are hitting over [117 million downloads/month](https://www.npmjs.com/package/@babel/core). It powers custom plugins and optimizations in varied scenarios like CSS-in-JS, GraphQL, or localization across enormous code bases.
+
+Babel also has become the intersection where practioners and language designers meet. We believe that it's become a vital part of the process used to test new ECMAscript proposals (hopefully, not with not too many unstable proposals in production!). This creates the ability for JavaScript developers to try new features and give feedback to the TC39 committee and ultimately influence how the language evolves. Babel doesn't represent any individual company's interest but hopefully the JavaScript users eagerness to participate in new ideas. Even if you or your company don't directly use Babel, you can still directly benefit from how it can help standardizing language features.
```

Comment:
```suggestion
Babel also has become the intersection where practioners and language designers meet. We believe that it's become a vital part of the process used to test new ECMAscript proposals (hopefully, not with too many unstable proposals in production!). This creates the ability for JavaScript developers to try new features and give feedback to the TC39 committee and ultimately influence how the language evolves. Babel doesn't represent any individual company's interest but hopefully the JavaScript users eagerness to participate in new ideas. Even if you or your company don't directly use Babel, you can still directly benefit from how it can help standardizing language features.
```